fabmukunzi/certificate-manager---Fabrice
# Task1 KAN-29: implement sidebar

#### PR description

- Implement sidebar

#### Ticket / Jira Link

- [KAN-29](https://task-force-pro-dccs.atlassian.net/jira/software/projects/KAN/boards/1?selectedIssue=KAN-29)

#### Checklist:


#### How To Test Changes

- Checkout to this branch `feature/task1-KAN-29-implement-sidebar
- Type `npm i` to install packages
- Open terminal and type `npm run dev` to run app
- Open your browser and navigate to `http://localhost:3000/` and navigate through the sidebar
- Open it in mobile view to check if it's responsive as well

#### Additional Comments

N/A

#### Screenshots:
![image](https://github.com/user-attachments/assets/0a8c9c22-ff4b-4608-9d20-72e4640a3342)
![image](https://github.com/user-attachments/assets/5cc8701f-cef0-4186-a4a3-28e63b47d936) ![image](https://github.com/user-attachments/assets/2b485984-0dd6-46a4-aabc-b312f14e0ae4)
